Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -199,6 +199,18 @@ class TelemetryWrapper {
         } else {
             GleanMetrics.Preferences.newTabExperience.set(NewTabAccessors.Default.rawValue)
         }
+        // Record `Home` setting, where Firefox Home is "Home", a custom URL is "other" and blank is "Blank".
+        let homePageSetting = NewTabAccessors.getHomePage(prefs)
+        switch homePageSetting {
+        case .topSites:
+            let firefoxHome = "Home"
+            GleanMetrics.Preferences.homePageSetting.set(firefoxHome)
+        case .homePage:
+            let customUrl = "other"
+            GleanMetrics.Preferences.homePageSetting.set(customUrl)
+        default:
+            GleanMetrics.Preferences.homePageSetting.set(homePageSetting.rawValue)
+        }
```

Comment:
Added tracking for Home Page here, where it looks like NewTab is also already tracked.

@dnarcese is newTab tracking correct? I think it might be better to track via `NewTabAccessors.getNewTabPage(prefs)` instead of `NewTabAccessors.Default.rawValue`

Reply:
In the end, its the same thing, right?

Reply:
While debugging, I noticed that it always returns `"HomePage"` regardless of whether New Tab is set to Firefox Home or Blank. It even returns "Home Page" when a custom Home doesn't exist in that list. 

By the way, I understood this to measure users who have set New Tab in Settings. If this isn't measuring that, you can ignore these comments 🙈